src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialRoleMappingTest.java

```diff
@@ -156,7 +173,8 @@ void shouldReturnFeePaidRoles(String setOffice, String expectedRoles, boolean wi
 
         //Execute Kie session
         List<RoleAssignment> roleAssignments =
-                buildExecuteKieSession(getFeatureFlags("sscs_wa_1_0", true));
+                buildExecuteKieSession(List.of(FeatureFlag.builder().flagName("sscs_wa_1_0").  status(true).build(),
```

Comment:
Can you turn both SSCS flags on in both sets of tests, create a common function if it helps: as this will reflect end state in PROD: and becomes easier to follow if version 3 & 4 are added etc...

Reply:
Updated

Reply:
Resolved in ada51b3009336d9d9a7feb18f849396d1bce795d.